### DISCUSSION TITLE
Shiny Jobs Battery Optimization on Android

### DISCUSSION THREAD

**Discussion by devonuto** [shinyorg/shiny#1044] [Q&A] [answered]
With scheduled jobs; is it recommended to turn off battery optimization? Are android jobs affected by this at all?

**Comment by aritchie** [comment, marked as answer]
I don't bother.  I don't rely on jobs for work that needs to be done on a schedule.  I believe battery optimization does slightly limit the workmanager, but you would need to look into the android docs to confirm.

**Reply by devonuto** [reply]
Thanks. Not important enough that the schedule is run at the same time all the time. Just that the job isn't killed altogether left in the background.
